kvmagent/kvmagent/plugins/ha_plugin.py

```
@@ -422,72 +508,184 @@ def get_vg_fencer_cmd(self, vg_uuid):
+            return res
+
+        if r.is_adding or r.get_renewal_last_result() == 1:
+            res.failed = False
+            return res
+
+        def is_heartbeat_timeout(timeout):
+            return abs(last_check - last_success) > timeout
+
+        try:
+            last_check = int(linux.monotime())
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)